From f066fc7a933dd04c2f293ea9944ddc096f9dfcca Mon Sep 17 00:00:00 2001
From: Jehan <jehan@girinstud.io>
Date: Sun, 13 May 2018 15:27:54 +0200
Subject: [PATCH] desktop: remove the double <releases> trick.

This was added in commits 447d9bbc56 and fb6328b9ad.
With it, the validation succeeds. Still I am actually not sure that is
proper appdata format. Also we have issues of metadata not shown for our
flatpak package, and I am wondering if this could be the issue.
So let's get rid of this hack.

Also temporarily commenting out the appdata unit test because it fails
without, because of a bug in appstream-util. This bug has been fixed
since, but there are no releases containing this fix yet.
See: https://github.com/hughsie/appstream-glib/issues/234
We'll reactivate the tests later.
---
 desktop/org.gimp.GIMP.appdata.xml.in.in | 2 --
 desktop/test-appdata.sh                 | 2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/desktop/org.gimp.GIMP.appdata.xml.in.in b/desktop/org.gimp.GIMP.appdata.xml.in.in
index 6a02df98dc..e547d1de34 100644
--- a/desktop/org.gimp.GIMP.appdata.xml.in.in
+++ b/desktop/org.gimp.GIMP.appdata.xml.in.in
@@ -124,9 +124,7 @@
         </ul>
       </description>
     </release>
-  </releases>
 
-  <releases>
     <release version="2.10.0~rc2" date="2018-04-17" type="development">
       <description>
         <_p>
diff --git a/desktop/test-appdata.sh b/desktop/test-appdata.sh
index d4bb50a9c1..b6735e692f 100755
--- a/desktop/test-appdata.sh
+++ b/desktop/test-appdata.sh
@@ -2,5 +2,5 @@
 
 # TODO: use validate-strict when the last errors for a strict validation
 # are fixed.
-appstream-util validate-relax ${GIMP_TESTING_ABS_TOP_BUILDDIR}/desktop/org.gimp.GIMP.appdata.xml && \
+# appstream-util validate-relax ${GIMP_TESTING_ABS_TOP_BUILDDIR}/desktop/org.gimp.GIMP.appdata.xml && \
 appstream-util validate-relax ${GIMP_TESTING_ABS_TOP_BUILDDIR}/desktop/gimp-data-extras.metainfo.xml
-- 
2.17.0

